ckanext/schema/dc_dataset.yaml

```diff
@@ -0,0 +1,321 @@
+    label: License
+    required: false
+    display_property: dct:license
+    form_snippet: license.html
+    help_text: License definitions and additional information can be found at http://opendefinition.org/.
+
+  - field_name: source
+    label: Source
+    required: false
+    display_property: dcat:accessURL
+    form_snippet: link.html
+    form_placeholder: http://example.com/project-url.html
+    help_text: A link to a project website related to the dataset.
+
+  # harvest_source not included here as it is added via ckanext-harvest plugin
+
+  - field_name: contact
+    label: Contact
+    required: false
+    display_property: dcat:contactPoint
+    form_snippet: text.html
+    form_placeholder: Primary contact for this dataset (name and email address or contact form link)
+
+  - field_name: data_applications
+    label: Data applications
+    required: false
+    help_text: List any known tutorials, tools, applications or publications associated with this dataset.
+    repeating_label: Application
+    repeating_subfields:
+    - field_name: category
+      label: Category
+      required: true
+      preset: select
+      choices:
+      - value: tutorials
+        label: Tutorials
+      - value: tools_and_applications
+        label: Tools and Applications
+      - value: publications
+        label: publications
+      
+    - field_name: title
+      label: Title
+      required: true
+      form_snippet: text.html
+
+    - field_name: url
+      label: URL
+      required: true
+      form_snippet: link.html
+
+    - field_name: author
+      label: Author
+      required: false
+      form_snippet: text.html
+
+  # On automated submissions, a SOURCES.md file will populate this field if present
+  - field_name: data_sources
+    label: Data sources
+    required: false
+    preset: markdown
+    display_property: prov:qualifiedPrimarySource
+    help_text: List or table of sources used to created this dataset.
+
+
+resource_fields:
+  - field_name: url
+    label: URL
+    required: true
+    preset: resource_url_upload
+    display_property: dcat:downloadURL
+
+  - field_name: name
+    label: Name
+    required: true
+    display_property: dct:title
+    form_snippet: text.html
+    form_placeholder: eg. 2021 UK Wholesale Electricity Prices
+
+  - field_name: description
+    label: Description
+    required: false
+    preset: markdown
+    display_property: dct:description
+    help_text: Add some useful notes about this data resource file.
+
+  # Will be inferred if empty
+  - field_name: file_format
+    label: File format
+    required: false
+    preset: resource_format_autocomplete
+    display_property: dct:format
+
+  # Could be inferred if empty
+  - field_name: file_size
+    label: File size
+    required: false
+    display_property: dct:byteSize
+    form_snippet: text.html
+    help_text: File size in bytes.
+
+  # If filled in this will take precedence over the datastore data dictionaries
```

Comment:
but maybe should be a verification step? Perhaps part of QA plugin

Reply:
Sorry, not sure if this is in relation to the `file_size` or `data_fields` property. Do you mean the comment about overriding the datastore data dictionaries?

Reply:
Apologies the context is not clear here. 
I meant to say if we use `data_fields` supplied by the publisher we may want to check them.

Reply:
Ah I see, OK